Header header
string node_name
float32 confidence
string description
